.github/workflows/build-msi-package.yml

```diff
@@ -25,6 +25,8 @@ jobs:
       - name: Build MSI
         run: dotnet build -c Release -p:Platform=x64
         working-directory: packaging/msi
+        env:
+          VCInstallDir: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise\\VC"
```

Comment:
With Visual Studio 2022 merge modules are installed in the VCInstallDir. Unfortunately this variable is not set in the GitHub actions shell.

Reply:
Should we put a comment on it? Would be nice to be able to computer this variable.

Reply:
Created https://github.com/zeroc-ice/ice/issues/3538